### DISCUSSION TITLE
llama 3.2 not working

### DISCUSSION THREAD

**Discussion by puru2901is** [lfnovo/open-notebook#26] [Q&A]
Getting following error
`Getting NotFoundError: Object with id model:hdjudawg1slroh3vkjln not found - model with id model:hdjudawg1slroh3vkjln not found`

Not sure how model id is coming hdjudawg1slroh3vkjln. I have run llama 3.2 with ollama

<img width="1645" alt="image" src="https://github.com/user-attachments/assets/45481fa5-dc2e-4995-b951-08d4f9600614">

**Comment by lfnovo** [comment]
this is the id for the model in the system database, it's not supposed to be your model name really. 
The issue is that it looks like you probably created and deleted a model and it kept the old version as the default. Just got to Models, make sure your model is there and then look at Default models and make sure the default models are set correctly to what you want. Let me know if this helps or we'll dive deeper.

**Comment by travelst614** [comment]
[]()

**Comment by travelst614** [comment]
- [ ] []()[]()_**@mgmanzella **_
